piqueserver/config/config.toml

```diff
@@ -0,0 +1,335 @@
+
+
+
+
+
+[info]
+
+# server name - displayed on master server
+name = "piqueserver instance"
+
+# messages sent when user joins server
+motd = [
+ "Welcome to %(server_name)s",
+ "Map: %(map_name)s by %(map_author)s",
+ "Game mode: %(game_mode)s",
+ "Server powered by piqueserver and buildandshoot.com"
+]
+
+# info displaying when user types the /help command
+# if not defined, /help will display a list of available commands
+# TODO: document string interpolation allowed
+help = [
+  "Server name: %(server_name)s", "Map: %(map_name)s by %(map_author)s",
+  "Game mode: %(game_mode)s",
+  "/streak    Shows how many kills in a row you got without dying",
+  "/intel     Tells you who's got the enemy intel",
+  "/votekick  Start a vote to temporarily ban a disruptive player",
+  "/time      Remaining time until forced map reset"
+]
+
+# list of messages to send when the user sends the /rules command
+rules = [
+  "Cheating isn't welcome. Griefing is frowned upon. Have fun!"
+]
+
+
+[tips]
+
+  # random lines from this list are shown at a specified frequency in-game
```

Comment:
Is the one-space indentation required?

Reply:
no that was unintentional

Reply:
actually i think the tips should be part of the info section anyway